octokit-project/src/main/java/org/example/calculator/Calculator.java

```diff
@@ -0,0 +1,15 @@
+package org.example.calculator;
+
+public class Calculator {
+
```

Comment:
[Severity: Minor] The main method currently contains hardcoded values for the variables a and b.
Consider allowing user input for the values of a and b to make the calculator more interactive.
Rationale: This aligns with the task goal of developing a console-based calculator application that performs basic arithmetic operations, enhancing usability.
Reference: Best practices for user input handling in console applications.